lib/humanize/words.rb

```diff
@@ -3,6 +3,8 @@ module Humanize
   WORDS = {
     :de => {
       :group_by => 1_000,
+      :infinity => 'Unendlichkeit', # needs verification
```

Comment:
Please use 'Unendlich', The rest is fine! Thx for the effort

Reply:
Thank you too @axelerator.